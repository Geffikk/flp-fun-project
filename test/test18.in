S,Y,Z,X
x
S
S->xxxx
S->XYZ
Y->xx
Z->x
X->x
