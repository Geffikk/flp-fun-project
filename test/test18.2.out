S,X,Y,Z,x',<xx>,<xxx>,<YZ>
x
S
S->x'<xxx>
x'->x
<xxx>->x'<xx>
<xx>->x'x'
S->X<YZ>
<YZ>->YZ
Y->x'x'
Z->x
X->x
